templates/github/.github/workflows/ci.yml.j2

```diff
@@ -43,6 +43,9 @@ jobs:
       {%- endif %}
   {%- endif %}
 
+  docs:
+    uses: "./.github/workflows/docs.yml"
+
```

Comment:
We _should_ keep this behind a flag, because a plugin outside of our control may want to do docs different.

Reply:
How do you think we should determine the docs-managed repos?
- fetching from the existing config file in pulp-docs?
- declaring a list here?
- other?

Reply:
Oh, I see your point. So the one point of authoritative information is the pulp-docs repository, yes?
Then we should take the information from there.